packages/react-native-reanimated/src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -289,7 +289,7 @@ export function createAnimatedComponent(
         ? (this._component as AnimatedComponentRef).getAnimatableRef?.()
         : this;
 
-      if (IS_WEB) {
+      if (shouldBeUseWeb()) {
```

Comment:
Not sure if some other places where we use `!IS_WEB`/`IS_WEB` checks should be updated as well. This change fixes the problem reported in the linked issue but I am not sure about other places where we use web check instead of `shouldBeUseWeb`.